tap_adroll/client.py

```diff
@@ -12,27 +12,39 @@
 
 class AdrollAuthenticationError(Exception):
     pass
-
-
 class AdrollClient():
-    def __init__(self, config_path, config):
-        token = {
-            'access_token': config['access_token'],
-            'refresh_token': config['refresh_token'],
-            'token_type': 'Bearer',
-            # Set expires_in to a negative number to force the client to reauthenticate
-            'expires_in': '-30'
-        }
-        extra = {
-            'client_id': config['client_id'],
-            'client_secret': config['client_secret']
-        }
+    def __init__(self, config_path, config, dev_mode = False):
+        self.dev_mode = dev_mode
         self.config_path = config_path
-        self.session = OAuth2Session(config['client_id'],
-                                     token=token,
-                                     auto_refresh_url=TOKEN_REFRESH_URL,
-                                     auto_refresh_kwargs=extra,
-                                     token_updater=self._write_config)
+        self.config = config
+        self.authenticate_request()
+
+    def authenticate_request(self):
+        if self.dev_mode :
+            self.session = requests.Session()
+            read_config = self._read_config()
+            try :
+                self._access_token = read_config['access_token']
+            except KeyError as err:
+                LOGGER.fatal("Unable to locate key %s in config",err)
```

Comment:
This log line is redundant because the [top level exception handler](https://github.com/singer-io/tap-adroll/blob/6c6ecd99ab2c6c94f149f2669f941eae2b456815/tap_adroll/__init__.py#L11) will print the error message raised on the next line

Reply:
Okay, removed the `LOGGER.fatal`.